ui/src/templates/components/CommunityTemplatesActivityLog.tsx

```diff
@@ -23,18 +30,10 @@ interface Resource {
   }[]
 }
 
-export class CommunityTemplatesActivityLog extends PureComponent<Props, State> {
-  state = {
-    stacks: [],
-  }
-
-  public async componentDidMount() {
+class CommunityTemplatesActivityLogUnconnected extends PureComponent<Props> {
+  public componentDidMount() {
     try {
-      // todo: move the stacks reponse into redux
-      const resp = await getStacks({query: {orgID: this.props.orgID}})
-
-      const stacks = (resp.data as any).stacks
-      this.setState({stacks})
+      this.props.fetchAndSetStacks(this.props.orgID)
```

Comment:
is there a reason we aren't using GetResources to fetch these?

Reply:
@ebb-tide yep, this is an MVP project that is dealing with a rapidly evolving API, while also trying to solve a chicken-and-egg product problem: this feature hasn't seen much use traditionally so we're not sure if we want to invest resources into it. But we're not sure if it hasn't seen much usage because we don't have good UX or because the feature itself isn't useful. So the focus of this is to learn what work we really need to do. It's not clear the ideal shape of the data or the how the UX of this should work, so while that's all in flight, I want to focus on iterating on the things are more visibly important to stakeholders (this is also a very high-visibility project, Evan has mentioned it in stand up this week).

My kind of long term vision if we decide to go with this is: making cleaning this up and making it conform to resources part of the work to replace old templates with community templates. Adding tests, tightening up the patterns, renaming confusing stuff, addressing all this tech debt I'm knowingly taking on, etc. But I don't want to do that work (which is super important and honestly my favorite kind of work) until we're sure it's worth the effort long term.

Reply:
👍👍